### PR TITLE
feat: add default support for async/await and generators

### PR DESCRIPTION
**Pull request by andregoncalvesdev**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -87,6 +88,17 @@ module.exports = (context, options) => {
         [require.resolve('babel-plugin-transform-object-rest-spread'), { useBuiltIns: true }]
     );
 
+    // Add async/await and generators support.
```
Comment (satazor) [inline index.js]: Remove the `.` at the end of the comment

File: README.md [hunk 2]
```diff
@@ -116,7 +120,6 @@ For instance, to have smaller bundles when using [recompose](https://github.com/
 ```
 
 
-
```
Comment (satazor) [inline README.md]: Please restore this new line.

**Comment by satazor** [comment]
@andregoncalvesdev After this gets merged, could you follow up with a PR on `react-with-moxy` disabling `regenerator `?

**Comment by andregoncalvesdev** [comment]
@satazor sure!

**Pull request merged**